app/controllers/answers_controller.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+class AnswersController < ApplicationController
+  skip_before_action :authenticate
+
+  # /answers?user_id=1 - lists all answer objects by user or question
+  # /answers?question_id=1&limit=2 - limits answer objects by votes
+  def index
+    return unless ensure_correct_param
+    answers = filter_answers
+    # limit_answers(answers, answer_limit) if params[:limit]
+    render_json(answers, :ok)
+  end
+
+  def ensure_correct_param
+    unless params[:user_id] || params[:question_id]
+      render_error('Missing parameter', :unprocessable_entity)
+      return false
```

Comment:
Just return is enough.

`return` implies `return nil`, and `nil` is falsey in Ruby